lib/fs.js

```diff
@@ -745,6 +745,14 @@ function mkdir(path, options, callback) {
                 validateMode(mode, 'mode', 0o777), recursive, req);
 }
 
+Object.defineProperty(mkdir, features, {
+  value: {
+    recursive: true
+  },
+  writable: false,
+  enumerable: false
```

Comment:
If these are intended to be non-configurable, then let's make that explicit.

That being said, I think they ought to be configurable. If someone *really* wants to block this feature detection, they can always just replace `mkdir`, etc. with their own versions that don't have the property. Rather than making them go through that, might as well just make it configurable.

e.g.

```js
const realMkdir = fs.mkdir;
fs.mkdir = (...args) => realMkdir(...args);
```